setup.py

```diff
@@ -59,6 +59,8 @@
         'invenio-sipstore>=1.0.0a1',
     ],
     'deposit': [
+        # Temporary pin to avoid collision with Flask-Security
```

Comment:
Looking which package is the one that depends on the Security one, as it seems that it's not just invenio-accounts that had it.